Dependencies/napi/napi-direct/CMakeLists.txt

```diff
@@ -72,10 +72,18 @@ if(NOT TARGET javascript_engine)
         endif()
     elseif(NAPI_JAVASCRIPT_ENGINE STREQUAL "Chakra")
         target_link_to_dependencies(javascript_engine INTERFACE "chakrart.lib")
-    elseif(NAPI_JAVASCRIPT_ENGINE STREQUAL "JavaScriptCore" AND UNIX AND NOT ANDROID AND NOT APPLE)
-        target_include_directories(napi PUBLIC "/usr/include/webkitgtk-4.0")
-        find_library(JSCORE_LIBRARY JavaScriptCore)
-        target_link_to_dependencies(javascript_engine INTERFACE ${JSCORE_LIBRARY})
+    elseif(UNIX AND NOT ANDROID AND NOT APPLE)
+        if(NAPI_JAVASCRIPT_ENGINE STREQUAL "JavaScriptCore")
+            target_include_directories(napi PUBLIC "/usr/include/webkitgtk-4.0")
+            find_library(JSCORE_LIBRARY JavaScriptCore)
+            target_link_to_dependencies(javascript_engine INTERFACE ${JSCORE_LIBRARY})
+        elseif(NAPI_JAVASCRIPT_ENGINE STREQUAL "V8")
+            target_include_directories(napi PUBLIC "/usr/include/v8")
+            find_library(V8_LIBRARY v8)
+            target_link_to_dependencies(javascript_engine INTERFACE ${V8_LIBRARY})
```

Comment:
Do we want this define in V8 for linux like the one in Windows?
`V8_COMPRESS_POINTERS`

I'm not sure it's better to split by OS all of a sudden here. Would it better to split against JS engine and then split by OS if needed?

Reply:
`V8_COMPRESS_POINTERS` might be great for Linux, issue is determining when building for 64bits targets with CMake. I take a note .